doc/release-notes.md

```diff
@@ -82,9 +82,17 @@ Low-level RPC changes
   * `getnetworkinfo`
   * `getwalletinfo`
   * `getmininginfo`
-
 - `dumpwallet` no longer allows overwriting files. This is a security measure
   as well as prevents dangerous user mistakes.
+- `getnewaddress` and `addmultisigaddress` RPC `account` named parameters have
```

Comment:
Do you want to keep an `address` alias around for this named parameter as well, along with a deprecation warning? Otherwise it should probably be marked as a breaking change.

Reply:
https://github.com/bitcoin/bitcoin/pull/11536#discussion_r149993521

> Do you want to keep an address alias around for this named parameter as well, along with a deprecation warning? Otherwise it should probably be marked as a breaking change.

Thanks. Somehow I missed this comment earlier. Added in 7ae567f87eb474dafd8b3cb1351a0bdfe9e41208